static.webpack.config.prod.js

```diff
@@ -99,7 +99,10 @@ const Config = merge.smartStrategy({
   plugins: [
     ...defaultPlugins,
     ...(debug ? debugPlugins : [])
-  ]
+  ],
```

Comment:
After discussing, we'd like to remove this config file and move the content of this config in line into the Component. There will need to be future work to tie this configuration into the actual functionality to avoid the current disconnect between this config and the actual values implemented in other components.